yahoo/meson.build

```diff
@@ -0,0 +1,18 @@
+yahoo_prpl = shared_library('yahoo',
+  'libyahoo.c',
+  'libyahoojp.c',
+  'libymsg.c',
+  'util.c',
+  'yahoo_aliases.c',
+  'yahoochat.c',
+  'yahoo_doodle.c',
+  'yahoo_filexfer.c',
+  'yahoo_friend.c',
+  'yahoo_packet.c',
+  'yahoo_picture.c',
+  'yahoo_profile.c',
+  'ycht.c',
+  include_directories: toplevel_inc,
+  dependencies: [libpurple_dep, glib_dep],
+  install_dir: libpurple_dep.get_variable('plugindir'),
+)
```

Comment:
`libyahoojp.c` is another plugin as you can't have multiple protocols in a single plugin in purple 2 and earlier.